source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -66,12 +61,32 @@ int main()
         std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
         std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;
 
-        // Convert from Zivid to Robot (Transformation Matrix --> any format)
-        const auto robotRotationRepresentations = zividToRobot(transformationMatrix);
-
-        // Convert from Robot to Zivid (any format --> Rotation Matrix)
-        const auto transformationMatrix2 =
-            robotToZivid(robotRotationRepresentations, transformationMatrix.translation());
+        /*
+         * Convert from Zivid to Robot (Transformation Matrix --> any format)
+         */
+        print_header("Convert from Zivid (Rotation Matrix) to Robot");
+        const Eigen::AngleAxisd axisAngle(transformationMatrix.linear());
+        std::cout << "AxisAngle:\n" << axisAngle.axis() << ", " << axisAngle.angle() << std::endl;
+        const auto rotation_vector = rotation_matrix_to_rotation_vector(transformationMatrix.linear());
+        std::cout << "Rotation Vector:\n" << rotation_vector << std::endl;
+        const Eigen::Quaterniond quaternion(transformationMatrix.linear());
+        std::cout << "Quaternion:\n" << quaternion.coeffs() << std::endl;
+        const auto rpy_list = rotation_matrix_to_list_of_roll_pitch_yaw(transformationMatrix.linear());
```

Comment:
Is there a reason for having this variable in this group of source lines? If not, I'd move it closer to where it's used.

Reply:
the function converts from rotation matrix to something else, so that belongs in that group of source lines. The output from this group is used in the next group of source lines. Recall that these groups used to be two functions, where the output of the first was used as input to the next. Then it was all stored in one class/struct